vars/testsInStage.groovy

```diff
@@ -25,6 +25,11 @@ boolean call() {
                              exit 0
                          fi
                          cd src/tests/ftest
-                         ./list_tests.py """ + parseStageInfo()['test_tag'],
+                         if [ -x list_tests.py ]
+                         then
+                             ./list_tests.py """ + parseStageInfo()['test_tag'] + """
+                         else
+                             ./launch.py --list """ + parseStageInfo()['test_tag'] + """
+                         fi""",
```

Comment:
The only other suggestion I would have made last time was a bit of DRYing such as:

```suggestion
                             cmd=('./list_tests.py')
                         else
                             cmd=('./launch.py' '--list')
                         fi
                         "${cmd[@]}"""" + parseStageInfo()['test_tag']
```

Reply:
My groovy isn't that good!

Reply:
Well, the optimization is actually in the shell script inside the groovy string, so no advanced groovy necessary.  It's all good, in any case.